src/autopas/containers/verletListsCellBased/verletLists/VerletListHelpers.h

```diff
@@ -84,7 +84,7 @@ class VerletListHelpers {
      * @param soa the soa
      * @param newton3 whether to use newton 3
      */
-    void SoAFunctor(SoA<SoAArraysType> &soa, bool newton3) override {
+    void SoAFunctor(SoAView<SoAArraysType> soa, bool newton3) override {
```

Comment:
have you tested if this is faster than passing references? Is move used here?

Reply:
I have talked about this here: https://github.com/AutoPas/AutoPas/pull/299#issuecomment-509968207

Summed up, I think it doesn't make any difference for nearly all use cases.

Do you have another opinion?

Reply:
I haven't measured what is faster, but I haven't measured any performance difference at least on my machine with gcc between the old implementation without SoAViews and with SoAViews, so I think regardless what method to pass the SoAView is used, measuring the time it needs is hard, because it is such a small impact.

Reply:
Ah I forgot I already read that comment^^
In theory, I agree. Would be interesting to verify though.

Reply:
But then again GCC doesn't manage to vectorize the functor (at least with N3). I think this should be fine as it is now.